packages/facility-server/app/sync/CentralServerConnection.js

```
@@ -167,6 +173,56 @@
     throw new Error(`Did not get a truthy response after ${maxAttempts} attempts for ${endpoint}`);
   }
 
+  async *stream(endpointFn) {
+    const waitTime = 10000; // retry once per 10 seconds
+    const maxAttempts = 6 * 60 * 12; // for a maximum of 12 hours
+    for (let attempt = 1; attempt <= maxAttempts; attempt++) {
+      const response = await this.fetch(endpointFn(), { method: 'GET', rawResponse: true });
+      const reader = response.body.getReader();
+
+      let partial = Buffer.alloc(0);
+      reader: while (true) {
```

Biome (1.9.4)

[error] Do not use the reader variable name as a label

The variable is declared here

Creating a label with the same name as an in-scope variable leads to confusion.

(lint/suspicious/noLabelVar)